sw/device/lib/dif/autogen/BUILD

```diff
@@ -0,0 +1,20 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+exports_files(["dif_uart_autogen.inc"])
+
+cc_library(
+    name = "uart",
+    srcs = [
+        "dif_uart_autogen.c",
+        "dif_uart_autogen.inc",
+        "//sw/device/lib/dif:dif_uart.h",
+    ],
+    linkstatic = True,
```

Comment:
You shouldn't need this: static linking will be enforced by cc_binary rules which include this as a dependency.